### PR TITLE
Remove the test `test_simulator_online_size`

### LINKED ISSUE #791: Remove the test `test_simulator_online_size`, since it is not testing Qiskit but the backend capabilities
The test `test_simulator_online_size` checks that a timeout error is triggered for big programs when `q_program.execute` is run. Currently, the test takes 1 minute to run, because that's the default of `_wait_for_job` (in `qiskit/backends/ibmq/ibmqjob.py`). This creates a request every 5 seconds, checking if for the status. This creates a lot of load in the backend and some time waste during testing.

I suggest the fix in #790 .

### PR DESCRIPTION
**Pull request by 1ucian0**
Fixes #791 

### Summary

This PR improves the way that the timeout in `test_simulator_online_size` is checked.

### Details and comments

There are two key changes in this PR. The first one is adding `timeout=float_info.min` as an argument of `q_program.execute`. This will warranty that a timeout is triggered. The second change is the simplification of the program since there is no need to send a big program to trigger the timeout.

The checking time is reduced from:
```
test_simulator_online_size (test.python.test_quantumprogram.TestQuantumProgram)
Test test_simulator_online_size. ... ok

----------------------------------------------------------------------
Ran 1 test in 60.914s
```
to
```
test_simulator_online_size (test.python.test_quantumprogram.TestQuantumProgram)
Test test_simulator_online_size. ... ok

----------------------------------------------------------------------
Ran 1 test in 2.702s
```

### CODE REVIEW

**Comment by atilag** [comment]
Hi Luciano, I don't think this test is about testing timeouts. Seems like is checking that the online simulator could actually execute a 31 qubit program. Having said that, it's true that I would not include that test as part of Qiskit, but as part of online simulator testsuit. @ewinston what do you think?

**Comment by 1ucian0** [comment]
Well... In that case, the test is incomplete :P Because `execute` just checks the status. We never know if that job finishes...

File: test/python/test_quantumprogram.py [hunk 1]
```diff
@@ -1132,16 +1132,16 @@ def test_simulator_online_size(self, qe_token, qe_url):
         """
```
Comment (delapuente) [inline test/python/test_quantumprogram.py]: I'm not sure about this, according to the test and parameters, the test wants to check if there is no response for huge circuits using the standard timeout. This patch changes the purpose of the test: now it is testing that it raises a `JobTimeoutError` for unmeeting waitings.

**Comment by delapuente** [comment]
As stated in #791, let's get rid of this.

**Pull request merged**